AppCenter/AppCenter/Internals/Storage/MSDBStorage.h

```diff
@@ -80,6 +80,16 @@ static NSString *const kMSSQLiteConstraintAutoincrement = @"AUTOINCREMENT";
  */
 + (NSDictionary *)columnsIndexes:(MSDBSchema *)schema;
 
+/**
+ * Create a table within an existing databse
+ *
+ * @param schema Schema describing the database.
+ * @param db database instance.
+ *
+ * @return operation status.
+ */
++ (NSUInteger)createTablesWithSchema:(MSDBSchema *)schema inOpenedDatabase:(void *)db;
```

Comment:
create table or tables?

Reply:
The schema lets us create multiple tables as needed. Will update the description

Reply:
Is the method used?

Reply:
Yes this is needed to create the user level table which happens later when the user logins and not during initialization